client-v2/src/lib/dnd/DropZone.tsx

```diff
@@ -17,10 +18,15 @@ type Props = OuterProps & ContextProps;
 
 interface State {
   isTarget: boolean;
+  isActive: boolean;
 }
 
+const DropZoneContainer = styled.div<{ isActive: boolean }>`
+  ${({ isActive }) => !isActive && 'pointer-events: none'}
```

Comment:
We could also choose not to render these elements, but this approach minimizes the impact on the DOM. Interested to hear any thoughts.

Reply:
This sounds reasonable to me!